CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.31] - 2020-07-08
```

Comment:
The release with this code in is 0.0.32 (https://github.com/whirli/whirli-client/releases)

Can you add an entry here for 0.0.32, saying it is fixes to active-referral ?